modules/install/upgrade/2.0.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+//get users not having a hashed password
```

Comment:
How is this script called?
I (kind of) like the idea, but also think that people can just request a new password using the "password forget" function.

Primarily, if we don't have an update function that calls this script, this change would either add a new script that is unused or delay the PR.